frontend/components/WaitingPages/PlayerWaiting.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import logo2 from '../public/logo2.png';
```

Comment:
You might wish to remove the '..' here. Upon using this component in a page, I think that the relative import path doesn't always work.

Reply:
Good catch, I will do just that.

Reply:
You can use `@` to refer to the package root.